iceoryx_posh/test/moduletests/test_mepoo_mempool.cpp

```diff
@@ -44,24 +46,63 @@ class alignas(32) MemPool_test : public Test
     iox::mepoo::MemPool sut;
 };
 
-TEST_F(MemPool_test, CTor)
+TEST_F(MemPool_test, MempoolCtorInitialisesTheObjectWithValuesPassedToTheCtor)
 {
+    char memory[8192];
+    iox::posix::Allocator allocator1{memory, 8192U};
```

Comment:
I find the suffix ``1`` a bit confusing when there isn't a second allocator or sut.